5-infrastructure/modules/base-env/variables.tf

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "env" {
+  type        = string
+  description = "The environment to prepare (dev/npd/prd)."
+}
+
+variable "business_unit" {
```

Comment:
This does not seem necessary to put in the SA name

Reply:
removing